.changelog/3410.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:improvement
```

Comment:
This will come later! I'm unsure of when this PR will actually land in a release and what the actual release notes might look like so I left it out specifically for this PR.